behind-the-veil-siteroot/imports/ui/components/pages/login/LoginPage.jsx

```diff
@@ -1,83 +1,106 @@
-                        <PersonIcon></PersonIcon>
+                <div className={"flex flex-col items-center gap-2.5 mt-12"}>
+                    <div className={"relative"}>
+                        <span className={"absolute left-2.5 top-1/2 -translate-y-1/2"}>
+                        <UserIcon className={"size-6 stroke-2 text-dark-grey"}/>
                         </span>
-                        <input type="text" placeholder="Username" style={{height: "30px", width: "300px", paddingLeft: '40px', outline: "1px solid lightgray", borderRadius: "3px"}} />
+                        <Input type="text"
+                               placeholder="Username"
+                               className={"pl-12 w-64 sm:w-96 lg:w-64 xl:w-96"}
+                               name="username"
+                               onChange={handleInputChange}/>
                     </div>
 
-                    <div style={{position: 'relative'}}>
-                        <span style={{position: 'absolute', left: '10px', top: '50%', transform: 'translateY(-50%)'}}>
-                        <KeyIcon></KeyIcon>
+                    <div className={"relative"}>
+                        <span className={"absolute left-2.5 top-1/2 -translate-y-1/2"}>
+                        <KeyIcon className={"size-6 stroke-2 text-dark-grey"}/>
                         </span>
-                        <input type="text" placeholder="Password" style={{height: "30px", width: "300px", paddingLeft: '40px', outline: "1px solid lightgray", borderRadius: "3px"}} />
+                        <Input type="password"
+                               placeholder="Password"
+                               className={"pl-12 w-64 sm:w-96 lg:w-64 xl:w-96 "}
+                               name="password"
+                               onChange={handleInputChange}
+                        />
                     </div>
-                    
-                    <div className="text-hyperlink-colour underline" style={{cursor: "pointer", marginLeft: "auto", marginRight: "75px"}}>Forgot password?</div>
-                </div>
-
-                <div style={{alignItems: "center", display: "flex", flexDirection: "column", gap: "10px", marginTop: "50px"}}>
-                    <Button className="bg-secondary-purple hover:bg-secondary-purple-hover" style={{width: "150px"}}>Sign in</Button>
-
-                    <NavLink to="/register"><Button style={{width: "150px"}}>Register</Button></NavLink>
 
+                    <div className="text-hyperlink-colour underline cursor-pointer ml-auto mr-[10%] right-0">
+                        Forgot password?
+                    </div>
                 </div>
 
+                <div className={"flex flex-col items-center gap-2.5 mt-12"}>
+                    <Button className="bg-secondary-purple hover:bg-secondary-purple-hover w-1/2 min-w-40"
+                            onClick={handleSubmit}>
```

Comment:
Nice work cleaning up the inline styles. This might be a bit out of scope of your ticket, but wanted to bring up:
We can wrap all of this in a form element (i.e. turn the outer div at line 57 and its corresponding closing tag into a `<form>` tag), and then apply type="submit" on this button, so that it acts like an actual login form (i.e. user can press enter to submit)

Reply:
wrapped in <form>, can press enter to submit